### ISSUE TITLE
particle flash cryptic error

### ISSUE THREAD

**Issue by kennethlimcp** [spark/particle-cli#123]
```
KENMBP:Desktop kennethlimcp$ particle flash photon-1 tinker
? Which type of device? Photon
attempting to flash firmware to your device photon-1
Flash device failed
[object Object]
```

**Comment by dmiddlecamp** [comment]
bump on this one, hitting this today.

**Comment by kennethlimcp** [comment]
again, not sure why though the binary is there 

```
KENMBP:platform-10 kennethlimcp$ particle flash elec2g tinker_electron_ken.bin
Including:
    tinker_electron_ken.bin
attempting to flash firmware to your device elec2g
Flash device failed
[object Object]
```

**Comment by emilyrose** [comment]
@kennethlimcp is this on Windows?

**Comment by kennethlimcp** [comment]
I am on Mac OSx 10.11 with CLI version `1.8.8`

**Comment by cguillemette** [comment]
I'm also having this issue.

**Comment by samf** [comment]
I'm on Mac OS 10.10.5 and am having this issue.

**Comment by fossum** [comment]
On Fedora 22 x64 having this issue... on initial inspection, it looks like a cloud issue. It was flashing just fine and then just started failing here.

**Comment by brycekahle** [comment]
Fixed with #163
